### PR TITLE
Adds permissions for calm adapter

### PR DESCRIPTION
**Pull request by kenoir**
In order to write to dynamo db!

### CODE REVIEW

File: terraform/iam.tf [hunk 1]
```diff
@@ -51,3 +51,22 @@ data "aws_iam_policy_document" "read_ingestor_q" {
     ]
   }
 }
+
+resource "aws_iam_role_policy" "ecs_calm_adapter_task" {
+  name = "ecs_task_calm_adapter_policy"
+  role = "${module.calm_adapter.role_name}"
+
+  policy = "${data.aws_iam_policy_document.allow_all_calm_db.json}"
```
Comment (alexwlchan) [inline terraform/iam.tf]: Missing a `terraform fmt`.

File: terraform/iam.tf [hunk 2]
```diff
@@ -51,3 +51,22 @@ data "aws_iam_policy_document" "read_ingestor_q" {
...
+}
+
+data "aws_iam_policy_document" "allow_all_calm_db" {
+  statement {
+    actions = [
+      "dynamodb:*",
```
Comment (alexwlchan) [inline terraform/iam.tf]: Let’s scope this a little tighter: looking at the [DynamoDB permissions](http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/api-permissions-reference.html), I think we only want:

* dynamodb:PutItem
* dynamodb:UpdateTable

Does that look right?

The adapter definitely doesn’t need create/destroy table access, ability to delete items, etc…

File: terraform/iam.tf (hunk 1 above)
Comment (kenoir) [inline terraform/iam.tf]: What am I missing? I can't seem to get `terraform fmt` to notice this.

File: terraform/iam.tf (hunk 2 above)
Comment (kenoir) [inline terraform/iam.tf]: 👍

File: terraform/iam.tf (hunk 1 above)
Comment (alexwlchan) [inline terraform/iam.tf]: It doesn’t? I was sort of expecting it to do:

```
name   = "ecs_task_calm_adapter_policy"
role   = "${module.calm_adapter.role_name}"

policy = "${data.aws_iam_policy_document.allow_all_calm_db.json}"
```

or similar. Guess not, never mind.

**Pull request merged**